source/user-manual/reference/tools/wazuh-keystore.rst

```diff
@@ -22,12 +22,26 @@ wazuh-keystore options
 +------------------------+---------------------------------------------------------+
 | **-v <VALUE>**         | Specifies the value associated with the key.            |
 +------------------------+---------------------------------------------------------+
+| **-vp <VALUE>**        | Specifies the path to a file with the value.            |
++------------------------+---------------------------------------------------------+
+
+Only one of the options **-v** or **-vp** can be used at a time.
+If none of them are present, the tool will read the value from the standard input.
+
 
 Example
 -------
 * Set indexer's username and password:
 
 .. code-block:: console
 
-    # /var/ossec/bin/wazuh-keystore -f indexer -k username -v admin
+    # echo 'admin' | /var/ossec/bin/wazuh-keystore -f indexer -k username
+    # echo 'admin' | /var/ossec/bin/wazuh-keystore -f indexer -k password
+
+
+* Alternate ways to set values:
+
+.. code-block:: console
+
+    # /var/ossec/bin/wazuh-keystore -f indexer -k username -vp /file/with/password
     # /var/ossec/bin/wazuh-keystore -f indexer -k password -v admin
```

Comment:
Is this ok? We're providing a "file with password" path to the "username" key. Do we mean?

```
   .. code-block:: console

      # /var/ossec/bin/wazuh-keystore -f indexer -k username -v admin
      # /var/ossec/bin/wazuh-keystore -f indexer -k password -vp /file/with/password
```

If not, we need to clarify what's in the file and what we are trying to achieve here.

Reply:
Thank you @javimed .
I'll clarify this section.

I made a mistake because in fact, it's valid to use any key with the `-vp` option. But only has sense to hide a password.